src/server/fiche-metier/useCases/consulterFicheMetier.useCase.ts

```diff
@@ -0,0 +1,11 @@
+import { Either } from '~/server/errors/either';
+import { FicheMétier } from '~/server/fiche-metier/domain/ficheMetier';
+import { StrapiFicheMetierRepository } from '~/server/fiche-metier/infra/strapiFicheMetier.repository';
+
+export class ConsulterFicheMetierUseCase {
+	constructor(private strapiFicheMetierRepository: StrapiFicheMetierRepository) {}
```

Comment:
En passant par l'interface, ça vaut le coup de renommer le champ aussi du coup
```suggestion
	constructor(private ficheMetierRepository: FicheMetierRepository) {}
```